great_expectations/rule_based_profiler/domain_builder/__init__.py

```diff
@@ -1,9 +1,18 @@
-from .domain_builder import DomainBuilder  # isort:skip
-from .table_domain_builder import TableDomainBuilder  # isort:skip
-from .column_domain_builder import ColumnDomainBuilder  # isort:skip
-from .simple_column_suffix_domain_builder import (  # isort:skip
+from great_expectations.rule_based_profiler.domain_builder.domain_builder import (  # isort:skip
```

Comment:
Thank you so much for turning these into absolute imports :)

Reply:
Thanks Will! I needed to change one to fix an issue and ended up changing them all.
@cdkini That sounds great to me! Absolute is more verbose but more readable and explicit about intent so I would recommend always using them unless there is a clear reason not to for a particular case (and recommended in [pep8](https://www.python.org/dev/peps/pep-0008/)). Let's do that in a separate focused PR and maybe add a check that they don't creep back in unless we explicitly flag them, maybe using something like https://pypi.org/project/flake8-absolute-import/#description. How about we add a backlog cleanup task for this?

Reply:
Added GREAT-612